src/Drupal/DrupalExtension/Context/MinkContext.php

```diff
@@ -533,6 +533,26 @@ public function assertSelectRadioById($label, $id = '') {
   }
 
   /**
+   * @When I select the radio button :label with the name :name
+   *
+   * @TODO convert to mink extension.
+   */
+  public function assertSelectRadioByName($label, $name) {
+    $element = $this->getSession()->getPage();
+    $radiobutton = $name ? $element->find('css', "[name='$name']") : $element->find('named', array('radio', $this->getSession()->getSelectorsHandler()->xpathLiteral($label)));
+    if ($radiobutton === NULL) {
+      throw new \Exception(sprintf('The radio button with "%s" was not found on the page %s', $name ? $name : $label, $this->getSession()->getCurrentUrl()));
+    }
+    $value = $radiobutton->getAttribute('value');
+// @TODO: Remove because it breaks Drupal style radio buttons if they have a description. The <label> is beside the <input>, so "getParent()"" returns the wrapper div.
+//    $labelonpage = $radiobutton->getParent()->getText();
+//    if ($label != $labelonpage) {
+//      throw new \Exception(sprintf("Button with id '%s' has label '%s' instead of '%s' on the page %s", $name, $labelonpage, $label, $this->getSession()->getCurrentUrl()));
+//    }
```

Comment:
I created this step because the assertSelectRadioById() method was failing when there is a description next to the radio button.

I am going to submit another PR with a fix for assertSelectRadioById